desc/compute/_metric.py

```diff
@@ -0,0 +1,605 @@
+from desc.backend import jnp
+
+from .data_index import register_compute_fun
+from .utils import cross, dot
+
+
+@register_compute_fun(
+    name="sqrt(g)",
+    label="\\sqrt{g}",
+    units="m^{3}",
+    units_long="cubic meters",
+    description="Jacobian determinant",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    function_of="rtz",
+    data=["e_rho", "e_theta", "e_zeta"],
+)
+def _sqrtg(params, transforms, profiles, data, **kwargs):
+    data["sqrt(g)"] = dot(data["e_rho"], cross(data["e_theta"], data["e_zeta"]))
+    return data
+
+
+@register_compute_fun(
+    name="|e_theta x e_zeta|",
+    label="|e_{\\theta} \\times e_{\\zeta}|",
+    units="m^{2}",
+    units_long="square meters",
+    description="2D jacobian determinant for constant rho surface",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    function_of="rtz",
+    data=["e_theta", "e_zeta"],
+)
+def _e_theta_x_e_zeta(params, transforms, profiles, data, **kwargs):
+    data["|e_theta x e_zeta|"] = jnp.linalg.norm(
+        cross(data["e_theta"], data["e_zeta"]), axis=1
+    )
+    return data
+
+
+@register_compute_fun(
+    name="|e_zeta x e_rho|",
+    label="|e_{\\zeta} \\times e_{\\rho}|",
```

Comment:
I'd prefer to keep it as is. e_zeta x e_rho is the outward normal (assuming positive jacobian), which is what we generally want and is consistent with the others which are also outward normals. Admittedly it doesn't matter since its just the norm, but i think it makes sense if you think of rho -> theta -> zeta as being cyclical